### ISSUE TITLE
Remove ChannelFinder base url with parameter

### ISSUE THREAD

**Issue by dylan171** [ChannelFinder/pyCFClient#14]
There is a hard-coded Channel Finder URL [here](https://github.com/ChannelFinder/pyCFClient/blob/89f5639335aa7ebcb910289e1f74089857180d62/channelfinder/ChannelFinderClient.py#L52).

I think this should be a configurable parameter?

**Comment by shroffk** [comment]
@mskinner5278 can you change the url to use the base url parameter

**Comment by dylan171** [comment]
@mskinner5278 I looked at the above changes to address this issue and I think it is incorrect.

I think it should be:
```
self.__session.mount(self.__baseURL, Ssl3HttpAdapter())
```
instead of 
```
self.__session.mount(BaseURL, Ssl3HttpAdapter())
```

**Comment by shroffk** [comment]
Addressed in PR #19
